src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRESTService.kt

```diff
@@ -244,25 +244,27 @@ class ZaakafhandelParametersRESTService @Inject constructor(
     }
 
     @GET
-    @Path("{zaakafhandelUUID}/documentTemplates")
+    @Path("{zaakafhandelUUID}/informatieObjectType/{informatieObjectTypeUUID}/documentTemplates")
```

Comment:
I personally prefer [spinal-case](https://blog.restcase.com/5-basic-rest-api-design-guidelines/) for URLs but let's stick with camelCase as this is the default in ZAC now. In this case the more correct Dutch spelling would be: `informatieobjectType`. This is also used elsewhere in the code base. The thing is a `type` of `informatieobject`.
In English it would be different btw: `informationObjectType`.

```suggestion
    @Path("{zaakafhandelUUID}/informatieobjectType/{informatieobjectTypeUUID}/documentTemplates")
```

Reply:
Fixed